src/components/transactions/transactions.js

```diff
@@ -5,12 +5,45 @@ import { BarChartComponent } from "../bar-chart/bar-chart";
 import { TableTransactionsComponent } from "../table-transactions/table-transactions";
 import { AddTransactionComponent } from "../add-transaction-dialog/add-transaction-dialog";
 
+import { TransactionListService } from "../../services/transaction-service";
+
 export class TransactionsComponent {
   constructor(mountPoint, props) {
     this.mountPoint = mountPoint;
     this.props = props;
   }
 
+  getStoredData() {
+    TransactionListService.get()
+      .then(list => this.showStoredTransactions(list))
+      .catch(e => console.error(`get transactions: ${e.message}`));
+  }
+
+  setStoredData(data) {
+    this.list.unshift(data);
+    TransactionListService.set(this.list)
+      .then()
```

Comment:
you do not need to write `.then()` if you do not have handler there. You can write `catch` directly